src/components/Signin.jsx

```diff
@@ -0,0 +1,268 @@
+import React,{useState} from 'react';
+import undraw_login from '../components/assets/images/undraw_Login.svg';
+import styled from 'styled-components';
+
+const Content = styled.div`
+  width: 70%;
+`;
+
+const Button = styled.button`
+  width:${props => props.width};
+  margin-left:${props=>props.marginLeft};
+`;
+
+const AuthLinkWrapper = styled.div`
+  margin:40px;
+  h2 { 
+    width:100%; 
+    text-align:center; 
+    border-bottom: 5px solid gray; 
+    line-height:0.1em; 
+    margin:10px 0 20px;
+  } 
+  h2 span { 
+    background:#fff; 
+    padding:0 10px; 
+  }
+
+`;
+
+const DisplayForLinks = styled.div`
+  display: inline;
+  list-style:none;
+  text-align: center;
+  width: 100%;
```

Comment:
Remove width margin padding justify properties from this. No use

Reply:
ok